servers/lib/src/cloudcmd/cloudcmd.ts

```diff
@@ -0,0 +1,31 @@
+import { INestApplication } from '@nestjs/common';
+import { ConfigService } from '@nestjs/config';
+import { Server } from 'socket.io';
+import * as cloudcmd from 'cloudcmd';
+
+const runCloudCMD = (app: INestApplication, optionsPath: string) => {
+  const filesPath = app.get(ConfigService).get<string>('LOCAL_PATH');
+
+  const { createConfigManager } = cloudcmd;
+  const configManager = createConfigManager({
```

Comment:
The `createConfigManager` is only a rename for cloudcmd. Why not directly use cloudcmd?
Also please think of a better name for configManager. Thanks.

Reply:
Ok thanks. Please keep the code snippet as it is.